packages/cloudflare_logpush/data_stream/dns_firewall/manifest.yml

```
@@ -143,7 +143,7 @@ streams:
         required: false
         show_user: true
         default: 5
-        description: The maximum number of SQS messages that can be inflight at any time.
+        description: Deprecated in agent version 8.16.0, this parameter is ignored if present, use number_of_workers instead. The maximum number of SQS messages that can be inflight at any time. 
```

```suggestion
        description: Deprecated in agent version 8.16.0, this parameter is ignored if present, use `Number of Workers` instead. The maximum number of SQS messages that can be inflight at any time. 
```

Users see the `title` of these options instead of `name`